packages/ui/primitives/document-flow/field-items-advanced-settings/dropdown-field.tsx

```diff
@@ -105,8 +105,12 @@ export const DropdownFieldAdvancedSettings = ({
           <Trans>Select default option</Trans>
         </Label>
         <Select
-          defaultValue={defaultValue}
+          value={defaultValue}
           onValueChange={(val) => {
+            if (!val) {
```

Comment:
the select component wouldn't show the defaultValue when passed dynamically via the API... this fixes it.